hw-6/controllers/auth.controller.ts

```diff
@@ -0,0 +1,21 @@
+import {Request, Response} from 'express';
+import * as jwt from 'jsonwebtoken';
+import {HTTP_ERROR, JWT_SECRET} from '../constants/http-errors.enum';
+
+function isValidUser(username: string, password: string): boolean {
+  return username === 'admin' && password === 'admin';
+}
+
+// eslint-disable-next-line import/prefer-default-export
+export const login = ({body}: Request, res: Response): void => {
+  const {username, password} = body;
+
+  if (isValidUser(username, password)) {
+    const payload = {user: 'admin'};
+    const token = jwt.sign(payload, JWT_SECRET, {expiresIn: 10});
```

Comment:
Just FYI, ideally, such types of configuration like the `expiresIn` value here might be stored out of the application code in either environment variables or environment config files.

If for some reason this value has to be changed due to security reasons or any other reason, we would have to make this change to propagate from development to production, which, depending on the project and client, the process could be very bureaucratic and time consuming (once in a project the client had 5 pre-production environment and we had to promote a change from one to another before reaching production).

Reply:
Good point. Fixing it!

Reply:
Fixed.